/**:
  ros__parameters:
    serial_port:
      name: "/dev/ttyUSB0"
      baud_rate: 115200

    BNO055:
      acc_factor: 100.0
      mag_factor: 16000000.0
      gyro_factor: 900.0
      grav_factor: 100.0
      data_frequency: 100.0
      make_calibration: false
      set_offset: true
      acc_offset: [65449, 9, 65413]
      mag_offset: [65468, 35, 65418]
      gyro_offset: [65534, 0, 0]
